script.js

```diff
@@ -1,7 +1,96 @@
 
 const hamburger = document.querySelector(".fa-bars");
 const menu = document.querySelector(".menu");
-const close = document.querySelector('fa-times');
+const close = document.querySelector('.fa-times');
+const myProjects = [
+    {
+        name: "Tonic",
+        preferedImage: "./assets/Snapshoot\ Portfolio.svg", 
+        description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+        languages: ['Canopy','backend', '2015'], 
+        tags: ['HTML','CSS','Javascript'], 
+        liveLink:'https://ernestkorir.github.io/my-portfolio/', 
+        sourceLink: 'https://github.com/ernestkorir/my-portfolio.git',
+        year: 2013
+    },
+    {
+        name: "Multi-Post Stories", 
+        preferedImage:"./assets/Snapshoot\ Portfolio2.svg",
+        description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+        languages: ['reactjs','frontend','2015'], 
+        tags: ['HTML','CSS','Javascript'], 
+        liveLink:'https://ernestkorir.github.io/my-portfolio/', 
+        sourceLink: 'https://github.com/ernestkorir/my-portfolio.git',
+        year: 2017
+    },
+    {
+        name: "Tonic", 
+        preferedImage: './assets/Snapshoot\ Portfolio3.svg',
+        description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+        languages: ['Backbonejs','typescript','2015'], 
+        tags: ['HTML','CSS','Javascript'],
+        liveLink:'https://ernestkorir.github.io/my-portfolio/', 
+        sourceLink: 'https://github.com/ernestkorir/my-portfolio.git',
+        year: 2019
+    },
+    {
+        name: "Multi-Post Stories",
+        preferedImage:"./assets/Snapshoot\ Portfolio4.svg",
+        description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+        languages: ['RDBMS','backend','2015'], 
+        tags: ['HTML','CSS','Javascript'], 
+        liveLink:'https://ernestkorir.github.io/my-portfolio/', 
+        sourceLink: 'https://github.com/ernestkorir/my-portfolio.git',
+        year: 2021
+    }
+    
+]
+
+const loadProjects = function() {
+    let projectsDiv = document.querySelector('section#projects');
+    let cardTemplate = projectsDiv.querySelector('#project-template');
```

Comment:
- [ ] When I click on project I get the popup with completely different topic and images. The same popup shows up on every click,. 

https://user-images.githubusercontent.com/77038610/195678879-662c4df5-506c-4e20-963b-0bb0ba771117.mp4